data/human/sales.txt

```diff
@@ -145,6 +145,56 @@ shipyard "Luxury Ships"
 	"Star Queen"
 	"Flivver"
 
+# For a few decades, New Tortuga has had advanced technology, Syndicate defectors, and a steady supply of poached employees. One of their top priorities has been technological advancement. They are only one planet though, so even with help from an advanced species, their success has been limited. Thus, this list is restricted low-to-mid-range tech from older companies. This is with the exception of the Syndicate, for which they already had access to top technology.
+shipyard "Independent (Korath-Allied)"
+	# Advanced Southern Pirates without the Osprey or Falcon:
+	"Modified Argosy"
+
+	# Commonly available from multiple sellers:
+	"Hauler"
+
+	# Common pirate fighter:
+	"Lance"
+
+	# Syndicate Advanced without the Container Transport or Arrow:
+	"Bulk Freighter"
+
+	# Non-modified versions of pirate-modified ships:
+	"Blackbird"
+	"Bastion"
+	"Behemoth"
+	"Bounder"
+
+	# Pirate modifications of standard ships:
+	"Marauder Bounder"
+	"Nighthawk (New Tortuga)"
+	"Valkyrie"
+	"Cutthroat"
+	"Mammoth"
```

Comment:
Does it make sense for the Mammoth to be buyable?
It kind of takes away from the FW mission's uniqueness.